app/helpers/application_helper.rb

```diff
@@ -292,4 +292,8 @@ def admin_config_label(method, content = nil)
   def admin_config_description(content)
     tag.p(content, class: "crayons-field__description") unless content.empty?
   end
+
+  def role_display_name(role)
+    role.name == "banned" ? "Suspended" : role.name.titlecase
+  end
```

Comment:
I pulled this out into a helper method to clean up the admin views a bit and make this code more reusable!